tee-worker/omni-executor/aa-contracts-client/src/omni_account_client.rs

```diff
@@ -75,6 +78,74 @@ impl<P: RpcProvider<Transaction = TransactionRequest>> OmniAccountClient<P> {
 			FixedBytes::<32>::abi_decode(&result).map_err(|_| error!("Could not decode owner"))?;
 		Ok(owner)
 	}
+
+	pub async fn add_passkey_signer(&self, pk: PasskeyPublicKey) -> Result<(), ()> {
+		let call_data = addPasskeySignerCall { pk }.abi_encode();
+		let tx = build_call_transaction(self.address, call_data);
+		self.rpc_client.send_transaction(tx).await.map_err(|_| ())?;
+		Ok(())
+	}
+
+	pub async fn remove_passkey_signer(&self, pk: PasskeyPublicKey) -> Result<(), ()> {
+		let call_data = removePasskeySignerCall { pk }.abi_encode();
+		let tx = build_call_transaction(self.address, call_data);
+		self.rpc_client.send_transaction(tx).await.map_err(|_| ())?;
+		Ok(())
+	}
+
+	pub async fn add_passkey_signer_from_string(&self, pubkey_str: &str) -> Result<(), String> {
+		let pk = parse_passkey_public_key(pubkey_str)?;
+		self.add_passkey_signer(pk)
+			.await
+			.map_err(|_| "Failed to add passkey signer".to_string())
+	}
+
+	pub async fn remove_passkey_signer_from_string(&self, pubkey_str: &str) -> Result<(), String> {
+		let pk = parse_passkey_public_key(pubkey_str)?;
+		self.remove_passkey_signer(pk)
+			.await
+			.map_err(|_| "Failed to remove passkey signer".to_string())
+	}
+}
+
+fn parse_passkey_public_key(pubkey_str: &str) -> Result<PasskeyPublicKey, String> {
```

Comment:
It depends on what format/encoding this `pubkey_str` has - I believe normally WebAuthn has a COSE encoded one, so probably we still need to change this and/or add some preprocessing.